crates/symbolicator/src/services/symbolication.rs

```diff
@@ -1997,47 +1976,78 @@ impl SymbolicationActor {
         cfi_results: Vec<CfiCacheResult>,
     ) -> Result<(SymbolicateStacktraces, MinidumpState), anyhow::Error> {
         let cfi_caches = CfiCacheModules::new(cfi_results.iter());
-
         let pool = self.spawnpool.clone();
         let diagnostics_cache = self.diagnostics_cache.clone();
         let lazy = async move {
             let spawn_time = std::time::SystemTime::now();
             let spawn_result = pool.spawn(
                 (
-                    procspawn::serde::Json(cfi_caches),
+                    procspawn::serde::Json(cfi_caches.clone()),
                     minidump.clone(),
                     spawn_time,
-                    options.compare_stackwalking_methods,
                 ),
-                |(cfi_caches, minidump, spawn_time, compare_stackwalking_methods)| {
-                    Self::procspawn_inner_stackwalk(
-                        cfi_caches,
-                        minidump,
-                        spawn_time,
-                        compare_stackwalking_methods,
-                    )
+                |(cfi_caches, minidump, spawn_time)| {
+                    Self::procspawn_inner_stackwalk(cfi_caches, minidump, Some(spawn_time), false)
                 },
             );
 
-            let (modules, stacktraces, minidump_state) = Self::join_procspawn(
+            let stackwalking_result_old = Self::join_procspawn(
                 spawn_result,
                 Duration::from_secs(60),
                 "minidump.stackwalk.spawn.error",
                 &minidump,
-                diagnostics_cache,
+                diagnostics_cache.clone(),
             )?;
 
+            metric!(timer("minidump.stackwalk.duration") = stackwalking_result_old.run_time, "method" => "old");
```

Comment:
what's the benefit of carrying the `run_time` to out here and record it here?  Why not remove a field from `stackwalking_result_old` and record the metric locally when you get it?

Reply:
Tbh I'm not sure why I did it that way originally ^^'